packages/eth-edge/src/services/transactions.service.ts

```diff
@@ -58,4 +58,9 @@ export class TransactionsService {
             where: [{ walletIn: In(addresses) }, { walletOut: In(addresses) }],
         });
     }
+
+    async delete(walletAdress: string): Promise<void> {
+        this.transactionsRepository.delete({ walletIn: walletAdress });
+        this.transactionsRepository.delete({ walletOut: walletAdress });
```

Comment:
Should be able to do this in 1 line as well, doesn't need to be 2 separate calls
```suggestion
        await this.transactionsRepository.delete({ walletIn: walletAdress });
        await this.transactionsRepository.delete({ walletOut: walletAdress });
```

Reply:
I wanted to ass a where clause but from what I can tell delete does not support it. The solution is using query builder which I don't think is a good option

Reply:
Can you let me know if you find anything on where clauses for delete?